script.js

```diff
@@ -391,16 +391,44 @@ portfoliobtn.forEach((item) => {
 
 //form validation
 
-const Form = document.querySelector('.form-layout');
+const form = document.querySelector('.form-layout');
 const email = document.getElementById('email-address');
-const ErrorMsg = document.querySelector('.validationMessage');
+const errorMsg = document.querySelector('.validationMessage');
+const fullName = document.getElementById('full-name');
+const userMessage = document.getElementById('message');
 
-Form.addEventListener('submit', (e)=> {
+form.addEventListener('submit', (e)=> {
   if (/[A-Z]/.test(email.value)) {
-    ErrorMsg.innerHTML = `Kindly change your email address to lowercase. <br> for example ${email.value.toLowerCase()}` ;
+    errorMsg.innerHTML = `Kindly change your email address to lowercase. <br> for example ${email.value.toLowerCase()}` ;
     e.preventDefault();
   }
-  })
+})
+
+function localSave() {
+  const setData = {
+    name: fullName.value,
+    Email: email.value,
+    Comment: userMessage.value,
+  };
```

Comment:
The form data for all inputs are stored in a single object, saved to a single local storage registry 🙌 Well done!